packages/lib/next-auth/auth-options.ts

```diff
@@ -138,6 +138,23 @@ export const NEXT_AUTH_OPTIONS: AuthOptions = {
         merged.emailVerified = user.emailVerified?.toISOString() ?? null;
       }
 
+      if (
+        (trigger === 'signIn' || trigger === 'signUp') &&
+        merged.emailVerified === null &&
+        account?.provider === 'google'
```

Comment:
Does this work? If it does that's amazing ❤️

Reply:
Pho sho 🙏

Reply:
I just realized that we're not using setting `identityProvider` on the user table to `GOOGLE` 

Should I just slide that in here? We need it set correctly for this PR: #709